### PR TITLE
[159] Create and Style the Customer Signup page using CSS

### PR DESCRIPTION
**Pull request by lorlahdey**
Capture19
Capture20
Capture21
Capture22

### CODE REVIEW

File: submissions/lawalLolade/css/contactUs.css [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    margin-top: 5%;
+    justify-content: space-between;
+}
+
+.contact-media li {
+    list-style: none;
+}
+
+.contact-media img {
+    width: 80%;
```
Comment (ToluAkin) [inline submissions/lawalLolade/css/contactUs.css:77]: Use `px` here @lorlahdey

File: submissions/lawalLolade/css/contactUs.css [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    font-family: 'Raleway', sans-serif;
+    font-size: 18px;
+    font-weight: bold;
+    line-height: 0.67;
+}
+
+   /* footer */
+footer{
+    background-color: #f765a3;
+    padding: 2% 3% 2%;
```
Comment (ToluAkin) [inline submissions/lawalLolade/css/contactUs.css:167]: Use `px` here @lorlahdey

File: submissions/lawalLolade/css/contactUs.css [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+    }
+    nav li {
+        display: block;
+        margin-bottom: 7%;
+    }
+    nav li:last-child {
+        margin-bottom: 0%;
+    }
+    nav{
+        margin-left: 0rem;
```
Comment (ToluAkin) [inline submissions/lawalLolade/css/contactUs.css:250]: There is no need to add unit if the value is 0 @lorlahdey

File: submissions/lawalLolade/css/productPage.css [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+
+.slidder {
+    margin: 2% 0;
+    padding: 0 2rem;
+}
+
+.slidder h2 {
+    text-align: center;
+    color: #5f5f5f;
+    font-family: 'Montserrat', sans-serif;
```
Comment (ToluAkin) [inline submissions/lawalLolade/css/productPage.css:53]: Set the font family in the body tag @lorlahdey

File: submissions/lawalLolade/css/productPage.css [hunk 5]
```diff
@@ -0,0 +1,355 @@
+* {
```
Comment (ToluAkin) [inline submissions/lawalLolade/css/productPage.css:1]: Change all % of margin and padding to px @lorlahdey

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Comment by lorlahdey** [comment]
ToluAkin

**Comment by lorlahdey** [comment]
@Ireti203

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**